ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -71,9 +71,10 @@ promotion:
   namespace: ocp-kni
 releases:
   candidate:
-    release:
-      channel: candidate
-      version: "4.8"
+    candidate:
```

Comment:
Is anyone going to be using this? Why add it back?

Reply:
Also, what's the difference between all three of these 4.9 release options?

Reply:
The nightly stream is the nightly builds. The other two describe more bleeding edge version of OCP picked up directly from the integration stream.

Docs: [Testing with ephemeral Openshift release](https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-ephemeral-openshift-release)